monitoring/bin/remove_monitoring_cluster.sh

```diff
@@ -64,6 +64,9 @@ log_verbose "Removing configmaps and secrets"
 kubectl delete cm --ignore-not-found -n $MON_NS -l sas.com/monitoring-base=kube-viya-monitoring
 kubectl delete secret --ignore-not-found -n $MON_NS -l sas.com/monitoring-base=kube-viya-monitoring
 
+log_verbose "Removing Kubelet"
```

Comment:
According to the Kubernetes doc "The kubelet is the primary "node agent" that runs on each node."  Given that, this message migh alarm some folks.  I suggest updating the message to "Removing kubelet service".